SUMMARY.md

```diff
@@ -124,7 +124,9 @@
 		* [POST Update Call](voice/methods/calls/postCallsCallId.md)
 		* [PUT Pause/Resume Recording](voice/methods/recordings/putCallsCallIdRecording.md)
 	* [Conferences](voice/methods/conferences/about.md)
-		* [POST Update Conference](voice/methods/conferences/postConferencesConferenceId.md)
+        	* [GET Conference Information](voice/methods/conferences/getConferencesConferenceId.md)
+        	* [POST Update Conference](voice/methods/conferences/postConferencesConferenceId.md)
+        	* [GET Member Information](voice/methods/conferences/getConferenceMember.md)
```

Comment:
I think following this order makes sense and seems to be already used in existing docs:
1) Create resource (`POST`)
2) Describe resource (`GET`)
3) Update resource (`POST`)

Edit: Actually, you're already following that; leaving for posterity only.

There's also a mix of tabs and spaces for indentation, which I think we should avoid:
```suggestion
		* [GET Conference Information](voice/methods/conferences/getConferencesConferenceId.md)
		* [POST Update Conference](voice/methods/conferences/postConferencesConferenceId.md)
		* [GET Member Information](voice/methods/conferences/getConferenceMember.md)
```

Reply:
The indentation was breaking it

Reply:
nope nvm